soc/arc/snps_nsim/Kconfig.defconfig.em11d

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2022 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NSIM_EM11D
+
+config CPU_EM4_FPUDA
```

Comment:
for EM11D (with caches enabled, as I figured from the nsim properties) the base should actually be EM6. EM4 doesn't have/enable caches).